src/emulator/mem/CMakeLists.txt

```diff
@@ -7,3 +7,4 @@ add_library(
 )
 
 target_include_directories(mem PUBLIC include)
+target_link_libraries(mem PUBLIC util spdlog)
```

Comment:
This can probably be removed until `mem` logs something.

Reply:
CMake and particular structure of this project isn't necessarily apparent to everyone working on this project... so I thought I'd do this for futureproofing, it doesn't hurt anything, only saves time for future development. Same for a couple other modules I've mentioned.